pkg/controllers/management/eks/eks_cluster_handler.go

```diff
@@ -0,0 +1,561 @@
+package eks
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"reflect"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	eksv1 "github.com/rancher/eks-operator/pkg/apis/eks.cattle.io/v1"
+	apimgmtv3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	apiprojv3 "github.com/rancher/rancher/pkg/apis/project.cattle.io/v3"
+	utils2 "github.com/rancher/rancher/pkg/app/utils"
+	"github.com/rancher/rancher/pkg/catalog/utils"
+	"github.com/rancher/rancher/pkg/clustermanager"
+	corev1 "github.com/rancher/rancher/pkg/generated/norman/core/v1"
+	mgmtv3 "github.com/rancher/rancher/pkg/generated/norman/management.cattle.io/v3"
+	projectv3 "github.com/rancher/rancher/pkg/generated/norman/project.cattle.io/v3"
+	"github.com/rancher/rancher/pkg/kontainer-engine/drivers/util"
+	"github.com/rancher/rancher/pkg/project"
+	"github.com/rancher/rancher/pkg/ref"
+	"github.com/rancher/rancher/pkg/systemaccount"
+	"github.com/rancher/rancher/pkg/types/config"
+	"github.com/rancher/rancher/pkg/wrangler"
+	wranglerv1 "github.com/rancher/wrangler/pkg/generated/controllers/core/v1"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/aws-iam-authenticator/pkg/token"
+)
+
+const (
+	globalNS            = "cattle-global-data"
+	systemNS            = "cattle-system"
+	eksAPIGroup         = "eks.cattle.io"
+	eksV1               = "eks.cattle.io/v1"
+	managementV3        = "management.cattle.io/v3"
+	eksOperatorTemplate = "system-library-rancher-eks-operator"
+	eksOperator         = "rancher-eks-operator"
+	localCluster        = "local"
+	creatorAnno         = "field.cattle.io/creatorId"
+	importedAnno        = "eks.cattle.io/imported"
+	awsAccessKey        = "amazonec2credentialConfig-accessKey"
+	awsSecretKey        = "amazonec2credentialConfig-secretKey"
```

Comment:
Can we pull some of these from more global const's ? Seems weird to redefine all as strings.

Reply:
changed the ones with exiting global vars